packages-exp/auth-exp/src/core/credentials/anonymous.test.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ProviderId, SignInMethod } from '@firebase/auth-types-exp';
+import { expect, use } from 'chai';
+import * as chaiAsPromised from 'chai-as-promised';
+import { testAuth } from '../../../test/mock_auth';
+import { Auth } from '../../model/auth';
+import { AnonymousCredential } from './anonymous';
+/*  */
+use(chaiAsPromised);
+
+describe('core/credentials/anonymous', () => {
+  let auth: Auth;
+  const credential = new AnonymousCredential();
```

Comment:
can you make this a variable and reinstantiate it every time in the `beforeEach()` block? I want to move away from tests ever having an object (other than a simple struct const) be shared across tests

Reply:
done